_includes/tutorials/splitting/confluent/markup/dev/ccloud-run-consumer.adoc

```diff
@@ -0,0 +1,37 @@
+Leave your original terminal running. To consume the output events from each of the topic, you'll need to open several new terminal windows. In each instance, the prompt will hang, waiting for more events to arrive. To continue studying the example, send more events through the input terminal prompt. Otherwise, you can `Control-C` to exit the process.
+
+First, to consume the events of drama films, run the following:
+
+```
+ccloud kafka topic consume drama-acting-events --from-beginning --value-format avro
```

Comment:
Isn't this text needed in this KT as well?  (I don't remember exactly but maybe you can pull it in as an include now?)

```
You will be prompted for the Confluent Cloud Schema Registry credentials as shown below, which you can find in the configuration/ccloud.properties configuration file. Look for the configuration parameter basic.auth.user.info, whereby the ":" is the delimiter between the key and secret.
Enter your Schema Registry API key:
Enter your Schema Registry API secret:
```

Reply:
oh man yes 100% that is needed. I reuse terminal windows which results in rarely getting this prompt. I've added it to the producer since that is the first time a user would encounter this. I think the single mention is enough.